src/shared-components/EventRecurrencePicker/EventRecurrencePicker.spec.tsx

```
@@ -0,0 +1,181 @@
+    customRecurrenceModalIsOpen,
+    hideCustomRecurrenceModal,
+  }: {
+    customRecurrenceModalIsOpen: boolean;
+    hideCustomRecurrenceModal: () => void;
+  }) =>
+    customRecurrenceModalIsOpen ? (
+      <div data-testid="customRecurrenceModal">
+        <button
+          data-testid="closeCustomModal"
+          onClick={hideCustomRecurrenceModal}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)